src/appleseed.shaders/src/appleseed/as_blackbody.osl

```diff
@@ -224,25 +238,29 @@ shader as_blackbody
         int as_maya_attribute_keyable = 0,
         int as_maya_attribute_hidden = 1,
         int as_blender_input_socket = 0,
-        int gafferNoduleLayoutVisible = 0
+        int gafferNoduleLayoutVisible = 0,
+        int as_max_param_id = 20
     ]],
     output closure color out_color = 0
     [[
         string as_maya_attribute_name = "outColor",
         string as_maya_attribute_short_name = "oc",
-        string widget = "null"
+        string widget = "null",
+        int as_max_param_id = 21
     ]],
     output closure color out_matte_opacity = 0
     [[
         string as_maya_attribute_name = "outMatteOpacity",
         string as_maya_attribute_short_name = "om",
-        string widget = "null"
+        string widget = "null",
+        int as_max_param_id = 21
     ]],
     output color out_blackbody_color = color(0)
     [[
         string as_maya_attribute_name = "outBlackbodyColor",
         string as_maya_attribute_short_name = "bbc",
-        string widget = "null"
```

Comment:
Multiple parameters with same ID?

Reply:
Output parameters get the same id because right now the are listed using single parameter in 3ds max.

Reply:
@usakhelo how can i find the IDs that max gives to the parameters?

Reply:
Hi @luisbarrancos ,
I added debug print statements to osl plugin to get ids to each parameter. Is that the question?
Also in normal plugins IDs are assigned in `ParamBlockDesc2` constructor.

Reply:
Thanks for the explanation @usakhelo.

Are you reviewing this PR @luisbarrancos?

Reply:
@dictoon i was taking a look yes, since we have https://github.com/appleseedhq/appleseed/pull/2426 draft on hold due to new parameters being added. I'll be updating the PR soon and once done perhaps @usakhelo can try the draft PR with Max. So far i was unable to get it working on Wine and unable to get GPU pass-through for a windows guest on KVM.